### DISCUSSION TITLE
How to not use githubusercontent.com?

### DISCUSSION THREAD

**Discussion by poikcue** [upptime/upptime#626] [Q&A] [answered]
Hello!
I'm currently using upptime as my status page. However, when I send the link to my status page to my friends, they tell me that "An error occurred" and within seconds of clicking the "Go to the homepage" button it will redirect to the error page.
I am currently located in China and after searching I found that there seems to be some problems connecting to githubusercontent.com from China. So, I was wondering if there is a way to not use githubusercontent.com or replace it with an accessible mirror URL?
Thanks!

**Comment by AnandChowdhary** [comment]
There's currently no configuration for this but I think it would be easy to add. Happy to accept a PR!

**Reply by poikcue** [reply]
Thank you for such a quick reply!
I was trying to find a solution yesterday. When I saw the gh-pages branch, I found that using these js files that start with English requires the use of raw.githubusercontent.com.
- client/index-.... .js 
- client/_number__..... .js (There seems to be two of this file. Only one of them contains this url.)

Unfortunately, when I try to manually modify raw.githubusercontent.com, it works fine. However, when after about a day it was automatically modified back to raw.githubusercontent.com by the bot.
I don't really know js, so I probably can't be of much help, so sorry!
(I use a translation tool because my English is not good, sorry!)

**Comment by GammaPi** [comment, marked as answer]
Fixed in https://github.com/upptime/status-page/pull/444
